streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -379,13 +379,12 @@ public void setUncaughtExceptionHandler(final Thread.UncaughtExceptionHandler un
     }
 
     /**
-     * Set the handler invoked when an {@link StreamsConfig#NUM_STREAM_THREADS_CONFIG internal thread}
+     * Set the handler invoked when an {@link StreamsConfig#NUM_STREAM_THREADS_CONFIG} internal thread
```

Comment:
I think this was actually correct as it was (and ditto for the above). One alternative suggestion:

```suggestion
     * Set the handler invoked when an internal {@link StreamsConfig#NUM_STREAM_THREADS_CONFIG stream thread}
```

Reply:
Updated. Thanks.